src/Dfc.CourseDirectory.Core/DataManagement/FileUploadProcessor.Apprenticeships.cs

```diff
@@ -214,14 +214,17 @@ protected async Task<UploadStatus> GetApprenticeshipUploadStatus(Guid apprentice
                 var rowIsValid = rowValidationResult.IsValid;
                 rowsAreValid &= rowIsValid;
 
+                int.TryParse(parsedRow.StandardCode, out int standardCode);
```

Comment:
This is required because the story to reject invalid standardcode/standard version has not been done yet. This is here to stop a sql exception trying to convert null to an integer.

Reply:
Testing shouldn't be using non-valid standard codes until that story is done. I would much rather this were `int.Parse`.

Reply:
changed